src/openforms/analytics_tools/migrations/0008_convert_ids_govmetric.py

```diff
@@ -0,0 +1,76 @@
+# Generated by Django 4.2.11 on 2024-04-12 08:43
+
+from django.db import migrations
+
+from openforms.translations.utils import get_language_codes
+
+
+def add_extra_source_id_govmetric(apps, schema_editor):
+    AnalyticsToolsConfiguration = apps.get_model(
+        "analytics_tools", "AnalyticsToolsConfiguration"
+    )
+
+    config = AnalyticsToolsConfiguration.objects.all().first()
```

Comment:
It's the same with `config = AnalyticsToolsConfiguration.objects.first()`, right?

Reply:
But can't we do what Vasileios suggested?

Reply:
This was done after the comment only in `remove_extra_source_id_govmetric` but yeah, I thought it wasn't worth another commit for this one as well.

Reply:
Oh! I had fixed it but I must have messed up in a rebase. I will fix it!